src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV2.scala

```diff
@@ -123,4 +130,19 @@ trait UserRoutesV2 extends SamUserDirectives with SamRequestContextDirectives {
       }
     }
 
+  private def getSamUserAttributes(samUser: SamUser, samRequestContext: SamRequestContext): Route =
+    get {
+      complete {
+        userService.getUserAttributes(samUser.id, samRequestContext).map(response => (if (response.isDefined) OK else NotFound) -> response)
+      }
+    }
+
+  private def patchSamUserAttributes(samUser: SamUser, samRequestContext: SamRequestContext): Route =
+    patch {
```

Comment:
this might be just me but i generally prefer put so that you can keep things idempotent

Reply:
This is PATCH because this endpoint will allow the caller to provide values only for the fields they want to update. A PUT would require the caller to specify all fields to store on the server, even if they're not changing.

Reply:
Right, that is sort of nice from our perspective, and there's only one field to really include on this call so far. I think this is fine just wanted to call out puts.

Reply:
I'm hoping that we start getting thurloe data into Sam soon so that this will be fleshed out!